db/migrate/20170609143035_create_permissions.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePermissions < ActiveRecord::Migration[5.1]
+  def change
+    # Add permissions table
+    create_table :permissions do |t|
+      t.belongs_to :user, index: true
+      t.string :action, null: false
+      t.string :subject_class, null: false
```

Comment:
Right, and could eventually be extended to include more classes by updating abilities.rb.